Makefile

```diff
@@ -124,7 +124,7 @@ firebase-public: frontend
 .PHONY: firebase-deploy firebase-deploy-prod firebase-emulate firebase-public
 
 # --- scripts ------------------------------------------------------------------
-SCRIPTS = firebase/deploy .github/scripts/app_token
+SCRIPTS = scripts/firebase-deploy .github/scripts/app_token
```

Comment:
oh, nice. turns out scripts has already been used for this purpose. perfect.
I think it's fine to keep the .github/scripts separate - wdyt?

Reply:
I would keep `.github/scripts` separate as a place for github actions only scripts.